go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/go-logr/logr v0.1.0
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.0
+	golang.org/x/text v0.3.7
```

Comment:
What about these changes? They seem a bit random in this PR :smile:

Reply:
They were necessary to fix linting issues https://app.circleci.com/pipelines/github/giantswarm/dns-operator-aws/842/workflows/1415a129-9be4-4805-bef0-e42c34cb371d/jobs/1927